.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Have you read Mocha's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md
+For more, check out the Mocha gitter space: https://gitter.im/mochajs/mocha
```

Comment:
how about instead of "gitter space" "Gitter chat room", since not everyone knows what Gitter is?